components/Reference/index.tsx

```diff
@@ -27,11 +27,10 @@ type CustomHeaderGroup = {
   className?: string
 } & HeaderGroup<IOpcode>
 
-// FIXME: Remove once https://github.com/comitylabs/evm.codes/issues/57 is done
 const DynamicFeeTooltip = () => (
   <span
     className="inline-block pl-2 text-gray-400 dark:text-black-400"
-    data-tip="Dynamic gas portion is not included yet and coming soon."
+    data-tip="Contains dynamic gas fee, expand the row to calculate it."
```

Comment:
How about something like "Has additional dynamic gas cost, expand to estimate it"?